src/components/benefitsDetailPage/query.ts

```diff
@@ -93,14 +114,11 @@ export const formatter: QueryFormatter<
     introText: getHtmlFromField(entity.field_intro_text_limited_html) || null,
     showTableOfContents: entity.field_table_of_contents_boolean ?? false,
     alert: entity.field_alert ? formatAlertBlock(entity.field_alert) : null,
-    // featuredContent:
-    //   entity.field_featured_content?.map((paragraph) =>
-    //     formatParagraph(paragraph)
-    //   ) || null,
-    // contentBlock:
-    //   entity.field_content_block?.map((paragraph) =>
-    //     formatParagraph(paragraph)
-    //   ) || null,
+    featuredContent:
+      entity.field_featured_content?.map((paragraph) =>
+        formatParagraph(paragraph)
+      ) || null,
```

Comment:
Do you know if doing the `getNestedIncludes()` with all those different paragraph types will result in an unhydrated entity if we miss one? Will we need to filter anything out here?